python/ray/scripts/scripts.py

```diff
@@ -246,7 +252,18 @@ def debug(address):
         sessions_data = sorted(
             sessions_data, key=lambda data: data["timestamp"], reverse=True
         )
-        table = [["index", "timestamp", "Ray task", "filename:lineno"]]
+        table = [
+            [
+                "index",
+                "timestamp",
+                "Ray task",
+                "filename:lineno",
+                "Node ID",
+                "Worker ID",
+                "Actor ID",
+                "Task ID",
```

Comment:
These should be ordered by usefulness. Should `Task ID` come earlier and `Node ID` come last?